.gitlab/rc_jobs/post_rc_tasks.yml

```diff
@@ -0,0 +1,19 @@
+# post_rc_build stage
+# Contains jobs that automate post-build Release Candidate tasks
+---
+update_rc_build_links:
+  stage: post_rc_build
+  rules:
+  - if: $RC_BUILD == "true"
+    when: always
+  needs:
+  - job: docker_trigger_internal
+    artifacts: false
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  script:
+    - source /root/.bashrc
+    - set +x
+    - export ATLASSIAN_PASSWORD=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.jira_read_api_token --with-decryption --query "Parameter.Value" --out text)
+    - export ATLASSIAN_USERNAME=robot-jira-agentplatform@datadoghq.com
```

Comment:
I think it would be worth re-enabling commands output here

Reply:
Good point, changed.